app/src/main/java/com/sunasterisk/weather/utils/ActivityUtils.kt

```diff
@@ -1,29 +1,48 @@
 package com.sunasterisk.weather.utils
 
+import android.app.Activity
+import android.content.Context
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentManager
 
-object ActivityUtils {
 
-    fun addFragmentToActivity(
-        fragmentManager: FragmentManager,
-        fragment: Fragment,
-        idRes: Int
-    ) {
-        fragmentManager.beginTransaction()
-            .setCustomAnimations(android.R.animator.fade_in, android.R.animator.fade_out)
-            .add(idRes, fragment)
-            .commit()
-    }
+fun AppCompatActivity.addFragmentToActivity(
+    fragmentManager: FragmentManager,
+    fragment: Fragment,
+    idRes: Int
+) {
+    fragmentManager.beginTransaction()
+        .setCustomAnimations(android.R.animator.fade_in, android.R.animator.fade_out)
+        .add(idRes, fragment)
+        .commit()
+}
+
+fun AppCompatActivity.replaceFragmentToActivity(
+    fragmentManager: FragmentManager,
+    fragment: Fragment,
+    idRes: Int
+) {
+   fragmentManager.beginTransaction()
+        .setCustomAnimations(android.R.animator.fade_in, android.R.animator.fade_out)
+        .replace(idRes, fragment)
+        .commit()
+}
+
+fun AppCompatActivity.removeFragmentToActivity(
+    fragmentManager: FragmentManager,
+    fragment: Fragment
+) {
+    fragmentManager.beginTransaction()
+        .setCustomAnimations(android.R.animator.fade_in, android.R.animator.fade_out)
+        .remove(fragment)
+        .commit()
+}
 
-    fun replaceFragmentToActivity(
-        fragmentManager: FragmentManager,
-        fragment: Fragment,
-        idRes: Int
-    ) {
-       fragmentManager.beginTransaction()
-            .setCustomAnimations(android.R.animator.fade_in, android.R.animator.fade_out)
-            .replace(idRes, fragment)
-            .commit()
+fun AppCompatActivity.setupToolbar(toolbar: Toolbar, titleToolbar: String) {
+    setSupportActionBar(toolbar)
+    supportActionBar?.apply {
+        title = titleToolbar
```

Comment:
thêm
`supportActionBar.setDisplayHomeAsUpEnabled(true)`
thì nó tự động hiển nút back trên toolbar
https://stackoverflow.com/questions/35810229/how-to-display-and-set-click-event-on-back-arrow-on-toolbar

Reply:
> thêm
> `supportActionBar.setDisplayHomeAsUpEnabled(true)`
> thì nó tự động hiển nút back trên toolbar
> https://stackoverflow.com/questions/35810229/how-to-display-and-set-click-event-on-back-arrow-on-toolbar
E muốn set nút back của e màu trắng á a